kubernetes/deployment/deployment.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       serviceAccountName: descheduler-sa
       containers:
         - name: descheduler
-          image: registry.k8s.io/descheduler/descheduler:v0.25.1
+          image: xiaobinimage/descheduler:v3
```

Comment:
may i ask why change the image repo?

Reply:
Sorry, forget to change test env back. I will update soon.